### PR TITLE
Added L0 tests and fixed some minor issues.

### PR DESCRIPTION
**Pull request by sundargs2000**
[This if](https://github.com/Azure/setup-helm/blob/d315e938e1f5b9cfdd84a5d79fa9c652c7c2787b/src/run.ts#L111) never evaluates to true. 
image

### CODE REVIEW

**Review by Josh-01 — CHANGES_REQUESTED** [review]

File: src/run.ts [hunk 1]
```diff
@@ -97,26 +97,26 @@ async function downloadHelm(version: string): Promise<string> {
...
 }
 
-function findHelm(rootFolder: string): string {
+export function findHelm(rootFolder: string): string {
     fs.chmodSync(rootFolder, '777');
     var filelist: string[] = [];
     walkSync(rootFolder, filelist, helmToolName + getExecutableExtension());
-    if (!filelist) {
-        throw new Error(util.format("Helm executable not found in path ", rootFolder));
+    if (filelist.length == 0) {
```
[the remark this answers is not in the document]
Comment (sundargs2000) [reply src/run.ts:111]: !filelist always evaluates to false as its an array.

File: tsconfig.json [hunk 2]
```diff
@@ -45,7 +45,7 @@
     // "typeRoots": [],                       /* List of folders to include type definitions from. */
     // "types": [],                           /* Type declaration files to be included in compilation. */
     // "allowSyntheticDefaultImports": true,  /* Allow default imports from modules with no default export. This does not affect code emit, just typechecking. */
-    "esModuleInterop": true                   /* Enables emit interoperability between CommonJS and ES Modules via creation of namespace objects for all imports. Implies 'allowSyntheticDefaultImports'. */
+    // "esModuleInterop": true                   /* Enables emit interoperability between CommonJS and ES Modules via creation of namespace objects for all imports. Implies 'allowSyntheticDefaultImports'. */
```
[the remark this answers is not in the document]
Comment (sundargs2000) [reply tsconfig.json:48]: `esModuleInterop` has something to do with importing commonjs modules, it is causing some errors while mocking in ts-jest. By default, `esModuleInterop` is false and we aren't using any such imports as well. Almost all of our actions have this set to false as well.

File: src/run.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (Josh-01) [reply src/run.ts:111]: It is an array now, but if it is set as null somewhere later, this will cause a NPE in the if statement

**Review by Josh-01 — APPROVED** [review]
Add the null check and we should be good

**Review by Josh-01 — APPROVED** [review]

**Pull request merged**